Package.swift

```diff
@@ -25,5 +25,5 @@ let package = Package(
         .testTarget(name: "MobiusNimbleTests", dependencies: ["MobiusNimble", "Quick"], path: "MobiusNimble/Test"),
         .testTarget(name: "MobiusTestTests", dependencies: ["MobiusTest", "Quick"], path: "MobiusTest/Test"),
     ],
-    swiftLanguageVersions: [.v4_2]
+    swiftLanguageVersions: [.v5]
```

Comment:
`[.v4_2, .v5]` for consistency?

Reply:
Why is that? This `Package.swift` will be used when targeting Swift 5, and `Package@swift-4.2.swift` when targeting Swift 4.2

Reply:
Bad brain moment there